### DISCUSSION TITLE
OpenAPI (fka Swagger) client & converter in python

### DISCUSSION THREAD

**Discussion by uurazzle** [univ-of-utah-marriott-library-apple/python-jamf#25] [Ideas]
Hi:

Here is an interesting item to checkout:

pyopenapi/pyswagger: An OpenAPI (fka Swagger) client & converter in python, which is type-safe, dynamic, spec-compliant.:

https://github.com/pyopenapi/pyswagger

**Comment by Honestpuck** [comment]
Are you thinking of reading endpoint details out of the Jamf Swagger files? It's a good idea, particularly since I believe we can read them off the server.

**Comment by uurazzle** [comment]
Yes, or at least investigate it as a possibility of speeding up and improving development on this project.

**Comment by magnusviri** [comment]
I think this is the same thing that I used to create the big dump of python files. I did it using the webpage https://editor.swagger.io/.

**Comment by magnusviri** [comment]
https://github.com/swagger-api/swagger-codegen
